addons/mail/data/mail_groups.xml

```diff
@@ -1,13 +1,19 @@
 <?xml version="1.0" encoding="utf-8"?>
 <odoo>
+<data>
```

Comment:
Should be in a noupdate=1 (like other groups, don't want to reset their naming at each -u)